server/liveblog/themes/themes.py

```diff
@@ -304,20 +307,32 @@ def on_updated(self, updates, original):
             self.publish_related_blogs(original)
 
     def on_delete(self, deleted_theme):
+        theme = deleted_theme['name']
         global_default_theme = get_resource_service('global_preferences').get_global_prefs()['theme']
         # raise an exception if the removed theme is the default one
-        if deleted_theme['name'] == global_default_theme:
+        if theme == global_default_theme:
```

Comment:
here you could let deleted_theme['name']. There is no need for previous assignments